packages/server/src/app.ts

```diff
@@ -196,7 +196,9 @@ export function initAppServices(config: MedplumServerConfig): Promise<void> {
     await initKeys(config);
     initBinaryStorage(config.binaryStorage);
     initWorkers(config);
-    initHeartbeat();
+    if (!config.heartbeatDisabled) {
+      initHeartbeat(config);
```

Comment:
Super minor preference to move this conditional into `initHeartbeat` so we can keep this function clean.